doc/staterecovery_sequencediagram.txt

```diff
@@ -1,16 +1,21 @@
 title State Recovery Procedure
 
 // TODO: remove walletd once it's stateless
+participant swapd
 participant walletd
 participant databased
 participant farcasterd
 participant cli
 
 == State Recovery Procedure
 entryspacing 0.8
+farcasterd -> walletd : launch
 farcasterd -> databased : launch
 cli -> databased : RetrieveAllCheckpointInfo
-databased -> cli : CheckpointList
+databased -> farcasterd : CheckpointList
```

Comment:
this is inconsistent with [main](https://github.com/farcaster-project/farcaster-node/blob/main/doc/staterecovery_sequencediagram.svg)